wercker.yml

```diff
@@ -28,6 +30,27 @@ ci:
         code: |
           bash <(curl -s https://codecov.io/bash)
 
+# build and test UI
+ci-ui:
+  box: node
+  steps:
+    - script:
+        name: install and build
+        code: |
+          cd ui
+          npm install
+          npm build
```

Comment:
I don't mind doing that since the dev compile process is quite different, will verify the code is able to be compiled for production.